README.md

```diff
@@ -1105,6 +1108,89 @@ However, the following rules of thumb are useful to keep in mind:
   Alternatively, the entire OrientDB class and all edges that may point to it may be omitted
   entirely from the GraphQL schema.
 
+## Query Cost Estimation
```

Comment:
The README.md lacks structure right now and there is no great place to put this section in, but this doesn't seem this is the right place. I would put it directly above the miscellaneous section for now.